doc/client_lua_api.md

```diff
@@ -117,10 +117,13 @@ The main Lua script. Running this script should register everything it
 wants to register. Subsequent execution depends on Luanti calling the
 registered callbacks.
 
-**NOTE**: Client mods currently can't provide textures, sounds, or models by
+**NOTE**: Client mods currently can't provide textures, or models by
 themselves. Any media referenced in function calls must already be loaded
 (provided by mods that exist on the server).
 
+However, sounds can be provided and used on a client machine if they are copied
```

Comment:
This is a different feature (sometimes referred to as a "sound pack") which may currently be undocumented (and I'm not sure whether we should document it in its current state).

Even if we decided to document this as-is however, I don't think the Client Lua API docs would be the right place for this.

Reply:
Sound packs could be an interesting feature if there was a way to distribute them officially and declare dependencies on them, but as this is not yet possible, I think that this mention is useful. Otherwise sound_play() is not very interesting, especially if you consider that you can't be sure of the game played and the ogg files bundled into it...